### PR TITLE
Fix some more quality issues

### PR DESCRIPTION
**Pull request by dunglas**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

### CODE REVIEW

File: src/GraphQl/Type/Definition/IterableType.php [hunk 1]
```diff
@@ -77,6 +77,8 @@ public function parseLiteral($valueNode)
 
     /**
      * @param StringValueNode|BooleanValueNode|IntValueNode|FloatValueNode|ObjectValueNode|ListValueNode $valueNode
+     *
+     *  @return mixed
```
Comment (meyerbaptiste) [inline src/GraphQl/Type/Definition/IterableType.php]: Too many spaces.

**Comment by norkunas** [comment]
You could also fix spacing [here](https://github.com/api-platform/core/blob/master/src/Bridge/Doctrine/Orm/Filter/DateFilter.php#L175).

**Comment by meyerbaptiste** [comment]
Done @norkunas.

**Comment by dunglas** [comment]
Thanks @meyerbaptiste

**Pull request merged**